### ISSUE TITLE
G0 - Pagination bar fails contrast test

### ISSUE THREAD

**Issue by jpinsent** [brharp/hjckrrh#120]
The pagination bar (as seen on the events & news listing pages) does not meet colour contrast requirements for the WCAG 2.0.
![contrast](https://cloud.githubusercontent.com/assets/12450480/9086238/20d4c04a-3b51-11e5-9890-bbcde3c282a7.PNG)
The white '1' on the blue background and the grey '...' on the white background both fail the AA test for small text.

**Comment by mmafe** [comment]
Fixed - grey dots are now #666666.

**Comment by jpinsent** [comment]
The grey ellipsis is passing now, but the white '1' over the blue background still does not meet accessibility standards. #FFFFFF over #428BCA does not meet colour contrast requirements on a AA level.

**Comment by mmafe** [comment]
Fixed in Hot Fix (Local Machine). Will be pushed to dev master this afternoon.

**Comment by jpinsent** [comment]
Just tested & looks good. It passes the colour contrast test now.
